src/Plugin/Field/FieldFormatter/MediaImageDisplayFormatter.php

```diff
@@ -139,18 +140,20 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
       '#options' => [
         'nothing' => $this->t('Nothing'),
 //        'content' => $this->t('Content'),
-        'media' => $this->t('Media')],
+        ($this->getFieldList($mediaFieldsList, 'link')) ?? 'media' => $this->t('Media')],
```

Comment:
$this->getFieldList($mediaFieldsList, 'link') should be put in a variable because it's call twice